.gitignore

```diff
@@ -16,3 +16,4 @@ web/app/yarn-error.log
 .golangci-lint*
 **/*.gogen*
 **/*.swp
+charts/*/charts/
```

Comment:
I thought git would gracefully handle symlinks?

Reply:
I ended up using `requirements.yaml` to refer to the local folder.

Reply:
ah I see you're using `requirements.yaml`